components/css/global/_footer.scss

```diff
@@ -429,76 +327,23 @@
 
         #accordion {
             .panel {
-                // float: left;
                 width: 20%;
                 padding: 0 15px;
-                // border: none;
-                // font-size: 14/16 * 1em;
-                // line-height: 20/12 * 1em;
-                // font-size: calc-toEm(14);
-                // line-height: calc-toEm(27);
-
-                // a {
-                //     padding: 8px 0 2px;
-                //     border-bottom: solid 1px #474747;
-                // }
-
-                // .panel-heading {
-                //     // display: block;
-                //     // padding: 0;
-                //     // background-color: transparent;
-                //     // // font-size: 18/14 * 1em;
-                //     // font-size: calc-toEm(21);
-                //     // line-height: 1em;
-                //
-                //     .panel-title {
-                //         // font-family: $font-champion-welter !important;
-                //         // padding-top: 0;
-                //         // padding-bottom: 5px;
-                //         // font-size: 1em;
-                //         // font-weight: 300;
-                //
-                //         // a {
-                //         //     cursor: default;
-                //         //     color: $color-campusGold;
-                //         //     border-bottom: none;
-                //         // }
-                //
-                //         // .fa {
-                //         //     display: none;
-                //         // }
-                //     }
-                // }
-                //
-                // .panel-collapse {
-                //     // display: block;
-                //     // height: auto !important;
-                // }
             }
 
             .motto {
                 float: left;
                 width: 20%;
                 padding: 40px 15px 15px;
-                // font-size: 20/16*1em;
                 font-size: calc-toEm(21.6);
 
                 .taglineContainer {
-                    // display: block;
-                    // position: absolute;
                     width: auto;
-                    // padding: 0;
-                    // margin: 0 auto;
 
                     .tagline {
                         margin-bottom:10px;
                         padding-bottom:71/202*100%;
-                        border-bottom: solid 1px #474747;
-                        // display: block;
-                        // width: 100%;
-                        // margin-right: auto !important;
-                        // margin-left: auto !important;
-                        // text-align: center !important;
+                        //border-bottom: solid 1px #474747;
```

Comment:
Did you mean to only comment this out rather than completely remove it?

Reply:
I imagine this was accidentally left in. I suppose it doesn't really matter when it comes to using minified styles since these comments are removed, but we can remove it anyways.